src/components/Product/ProductDetailImages.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { fireEvent, render } from '@testing-library/react';
+import ProductDetailImages from './ProductDetailImages';
+
+const images = [
+    {
+        src: '/products/images/00dca6e72ae5506507e7270ea7a62792.jpeg',
+        name: 'Apple MacBook Pro 14" Laptop Front View',
+    },
+    {
+        src: '/products/images/e38be691b33159f714e391102d782ef3.jpeg',
+        name: 'Apple MacBook Pro 14" Laptop Top View',
+    },
+    {
+        src: '/products/images/376bcac4f0be5a12b63d617c0e6348fe.jpeg',
+        name: 'Apple MacBook Pro 14" Laptop Side Open View',
+    },
+];
+
+describe('ProductDetailImages', () => {
+    it('Loads with all images', () => {
+        const { getByLabelText } = render(
+            <ProductDetailImages images={images} maxQuantity={123} />
+        );
+        const imagesInDOM = document.getElementsByTagName('img');
+        expect(imagesInDOM).toHaveLength(images.length + 1); // loads all images
+        [...imagesInDOM].forEach((image, i) => {
+            if (i !== 0) expect(image.src).toContain(images[i - 1].src);
+            else expect(image.src).toContain(images[0].src);
```

Comment:
should this be images[0] or images[i]?

Reply:
Both works, but `images[0]` gives it more specificity, and readers will avoid an extra step in trying to understand what's going on here.